### PR TITLE
Openshift e2e changes

### PR DESCRIPTION
**Pull request by mumutyal**

### CODE REVIEW

File: block-storage-attacher/tests/e2e/conf/developer.conf [hunk 1]
```diff
@@ -43,13 +43,14 @@ IMAGE_REGISTRY=stg.icr.io
...
 DRIVER_BUILD=$PLUGIN_BUILD
+OPENSHIFT_INSTALL=$OPENSHIFT_INSTALL
 
 # Helm chart install
 KUBECONFIG=$KUBECONFIG			# Read from environment
 HELM_CHART_REPO=$HELM_CHART_REPO
 HELM_CHART=$GOPATH/src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher/helm/ibm-block-storage-attacher/
 
-PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/portworx
+PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/community/portworx
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/developer.conf]: This helm chart path is wrong I guess. The current charts of portworx are at `https://github.com/IBM/charts/tree/master/community/portworx`. So need to use this location.

File: block-storage-attacher/tests/e2e/conf/prod-au-syd.conf [hunk 2]
```diff
@@ -41,12 +41,15 @@ USER_NAMESPACE=ibm
...
+#Openshift Install flag
+OPENSHIFT_INSTALL=$OPENSHIFT_INSTALL
+
 # Helm chart install
 KUBECONFIG=$KUBECONFIG			# Read from environment(incase of dev executions)
 HELM_CHART_REPO=$HELM_CHART_REPO	
 HELM_CHART=$GOPATH/src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher/helm/ibm-block-storage-attacher/
 
-PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/portworx
+PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/community/portworx
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/prod-au-syd.conf]: This helm chart path is wrong I guess. The current charts of portworx are at https://github.com/IBM/charts/tree/master/community/portworx. So need to use this location.

File: block-storage-attacher/tests/e2e/conf/prod-eu-de.conf [hunk 3]
```diff
@@ -41,12 +41,15 @@ USER_NAMESPACE=ibm
...
+#Openshift Install flag
+OPENSHIFT_INSTALL=$OPENSHIFT_INSTALL
+
 # Helm chart install
 KUBECONFIG=$KUBECONFIG			# Read from environment(incase of dev executions)
 HELM_CHART_REPO=$HELM_CHART_REPO	
 HELM_CHART=$GOPATH/src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher/helm/ibm-block-storage-attacher/
 
-PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/portworx
+PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/community/portworx
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/prod-eu-de.conf]: This helm chart path is wrong I guess. The current charts of portworx are at https://github.com/IBM/charts/tree/master/community/portworx. So need to use this location.

File: block-storage-attacher/tests/e2e/conf/prod-eu-gb.conf [hunk 4]
```diff
@@ -37,12 +37,15 @@ USER_NAMESPACE=ibm
...
+#Openshift Install flag
+OPENSHIFT_INSTALL=$OPENSHIFT_INSTALL
+
 # Helm chart install
 KUBECONFIG=$KUBECONFIG			# Read from environment(incase of dev executions)
 HELM_CHART_REPO=$HELM_CHART_REPO	
 HELM_CHART=$GOPATH/src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher/helm/ibm-block-storage-attacher/
 
-PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/portworx
+PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/community/portworx
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/prod-eu-gb.conf]: This helm chart path is wrong I guess. The current charts of portworx are at https://github.com/IBM/charts/tree/master/community/portworx. So need to use this location.

File: block-storage-attacher/tests/e2e/conf/prod-jp-tok.conf [hunk 5]
```diff
@@ -41,12 +41,15 @@ USER_NAMESPACE=ibm
...
+#Openshift Install flag
+OPENSHIFT_INSTALL=$OPENSHIFT_INSTALL
+
 # Helm chart install
 KUBECONFIG=$KUBECONFIG			# Read from environment(incase of dev executions)
 HELM_CHART_REPO=$HELM_CHART_REPO	
 HELM_CHART=$GOPATH/src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher/helm/ibm-block-storage-attacher/
 
-PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/portworx
+PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/community/portworx
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/prod-jp-tok.conf]: This helm chart path is wrong I guess. The current charts of portworx are at https://github.com/IBM/charts/tree/master/community/portworx. So need to use this location.

File: block-storage-attacher/tests/e2e/conf/prod-us-east.conf [hunk 6]
```diff
@@ -49,12 +49,15 @@ USER_NAMESPACE=ibm
...
+#Openshift Install flag
+OPENSHIFT_INSTALL=$OPENSHIFT_INSTALL
+
 # Helm chart install
 KUBECONFIG=$KUBECONFIG			# Read from environment(incase of dev executions)
 HELM_CHART_REPO=$HELM_CHART_REPO	
 HELM_CHART=$GOPATH/src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher/helm/ibm-block-storage-attacher/
 
-PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/portworx
+PORTWORX_HELM_CHART=$GOPATH/portworx/helm/community/portworx
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/prod-us-east.conf]: This helm chart path is wrong I guess. The current charts of portworx are at https://github.com/IBM/charts/tree/master/community/portworx. So need to use this location.

File: block-storage-attacher/tests/e2e/conf/prod-us-south.conf [hunk 7]
```diff
@@ -47,12 +47,16 @@ PLUGIN_IMAGE=ibmcloud-block-storage-attacher
...
+OPENSHIFT_INSTALL=$OPENSHIFT_INSTALL
+
+
 # Helm chart install
 KUBECONFIG=$KUBECONFIG			# Read from environment(incase of dev executions)
 HELM_CHART_REPO=$HELM_CHART_REPO	
 HELM_CHART=$GOPATH/src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher/helm/ibm-block-storage-attacher/
 
-PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/portworx
+PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/community/portworx
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/prod-us-south.conf]: This helm chart path is wrong I guess. The current charts of portworx are at https://github.com/IBM/charts/tree/master/community/portworx. So need to use this location.

File: block-storage-attacher/tests/e2e/conf/stage.conf [hunk 8]
```diff
@@ -46,11 +46,15 @@ USER_NAMESPACE=contsto2
...
+
+#Openshift Install flag
+OPENSHIFT_INSTALL=$OPENSHIFT_INSTALL
+
 # Helm chart install
 KUBECONFIG=$KUBECONFIG
 HELM_CHART_REPO=$HELM_CHART_REPO
 HELM_CHART=$GOPATH/src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher/helm/ibm-block-storage-attacher/
-PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/portworx
+PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/community/portworx
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/stage.conf]: This helm chart path is wrong I guess. The current charts of portworx are at https://github.com/IBM/charts/tree/master/community/portworx. So need to use this location.

File: block-storage-attacher/tests/e2e/e2e.sh [hunk 9]
```diff
@@ -30,6 +30,10 @@ SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"/scripts
 if [[ $TEST_BLUEMIX_LOGIN == "true" ]]; then
         echo "Bluemix Login DOne"
 	bx_login
+        if [ $ARMADA_REGION == "us-south" ]; then
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/e2e.sh]: can you explain what this check does specific to us-south? how its handled for other regions.

File: block-storage-attacher/Dockerfile [hunk 10]
```diff
@@ -12,6 +12,8 @@ LABEL git-remote-url=${git_remote_url}
 LABEL build-date=${build_date}
 LABEL jenkins-build-number=${jenkins_build_number}
 
+RUN apk add --update
+RUN apk upgrade
```
Comment (akgunjal) [inline block-storage-attacher/Dockerfile]: i see these above lines are present in master currently. So check and remove from the PR

File: block-storage-attacher/tests/e2e/conf/prod-au-syd.conf (hunk 2 above)
Comment (mumutyal) [reply block-storage-attacher/tests/e2e/conf/prod-au-syd.conf]: The path is correct . I have verified it

File: block-storage-attacher/tests/e2e/conf/prod-eu-de.conf (hunk 3 above)
Comment (mumutyal) [reply block-storage-attacher/tests/e2e/conf/prod-eu-de.conf]: No . The path I gave is proper

File: block-storage-attacher/tests/e2e/e2e.sh (hunk 9 above)
Comment (mumutyal) [reply block-storage-attacher/tests/e2e/e2e.sh]: This is required for staging only

File: block-storage-attacher/Dockerfile (hunk 10 above)
Comment (mumutyal) [reply block-storage-attacher/Dockerfile]: I think if I remove again merge confilct wil raise. let it be

**Pull request merged**